addons/web/static/src/legacy/js/libs/jquery.js

```diff
@@ -147,6 +147,31 @@ $.fn.extend({
         }
         return $el;
     },
+    /**
+     * Adapt the given css property by adding the size of a scrollbar if any.
+     * Limitation: only works if the given css property is not already used as
+     * inline style for another reason.
+     *
+     * @param {boolean} [add=true]
+     * @param {boolean} [isScrollElement=true]
+     * @param {string} [cssProperty='padding-right']
+     */
+    compensateScrollbar(add = true, isScrollElement = true, cssProperty = 'padding-right') {
+        for (const el of this) {
+            // Compensate scrollbar
+            el.style.removeProperty(cssProperty);
+            if (!add) {
+                return;
+            }
+            const scrollableEl = isScrollElement ? el : $(el).parent().closestScrollable()[0];
+            const style = window.getComputedStyle(el);
+            const borderLeftWidth = parseInt(style.borderLeftWidth.replace('px', ''));
+            const borderRightWidth = parseInt(style.borderRightWidth.replace('px', ''));
```

Comment:
Beware that the computed `borderWidth` properties are not always integers if the page is zoomed.
See <https://github.com/odoo/odoo/pull/119084>.
The current best approximation is `Math.ceil(parseFloat(style.borderLeftWidth))`.

Reply:
> Beware that the computed `borderWidth` properties are not always integers if the page is zoomed. See #119084. The current best approximation is `Math.ceil(parseFloat(style.borderLeftWidth))`.

Thank you, I made the change even though I'm not sure it would really cause an issue in this case.